rootfs/templates/kops/default.yaml

```diff
@@ -18,37 +18,46 @@ spec:
   api:
     loadBalancer:
       type: Public
+  {{- if bool (getenv "KOPS_AUTHORIZATION_RBAC_ENABLED") }}
+  kubeAPIServer:
+    authorizationMode: RBAC,AlwaysAllow
```

Comment:
make this an env that defaults to `RBAC,AlwaysAllow`

Reply:
done